src/System.Management.Automation/help/HelpProvider.cs

```diff
@@ -246,9 +246,7 @@ internal string GetDefaultShellSearchPath()
         /// <returns>true if supported,false otherwise.</returns>
         internal bool AreSnapInsSupported()
```

Comment:
I think this method should be removed, and update the two call sites in `HelpProviderWithFullCache.cs`:
`if (!this.CacheFullyLoaded || AreSnapInsSupported())` ==> `if (!this.CacheFullyLoaded)`

Reply:
will update and remove no longer relevant comment